macros/customizeLaTeX.pl

```diff
@@ -0,0 +1,84 @@
+=head1 NAME
+
+customizeLaTeX.pl - Defines default LaTeX constructs for certain mathematical
+                    ideas.
+
+=head1 DESCRIPTION
+
+The functions are loaded by default.  Any/all can be overridden 
+in your course's PGcourse.pl
+=cut
+
+##### Set theory macros
+sub set_minus{ 
+	#return "\\setminus";
+	return '-';
+};
+
+#####  Logic macros
+sub negate { 
+	return "\\sim";
```

Comment:
`\sim` is a LaTeX binary relation, and so something like `x\vee\sim y` doesn't look right. There will be too much space between the `\sim` and the `y`. I think (but am not sure) that the "right" thing to use is `\mathop{\sim}`.

Reply:
No, `\mathop{\sim}` will make it act like `\sum`.  You want it to act like `-` so use `\mathbin{\sim}`